12.customization-minimizable-web-chat Sample Doesn't Build

Fresh installs of everything and the latest code from the repo produced this error:

```
Failed to compile

./node_modules/react-say/lib/SayPrimitive.js
Module not found: Can't resolve 'babel-runtime/core-js/object/get-prototype-of' in 'C:\Code\BotFramework-WebChat\samples\12.customization-minimizable-web-chat\node_modules\react-say\lib'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix [#1423](https://github.com/Microsoft/BotFramework-WebChat/issues/1423). Added sample for hosting WebChat in Angular, by [@omarsourour](https://github.com/omarsourour) in PR [#1813](https://github.com/Microsoft/BotFramework-WebChat/pull/1813)
 - Fix [#1767](https://github.com/Microsoft/BotFramework-WebChat/issues/1767). Remove `cursor: pointer` from buttons, by [@corinagum](https://github.com/corinagum) in PR [#1819](https://github.com/Microsoft/BotFramework-WebChat/pull/1819)
 - Fix [#1774](https://github.com/Microsoft/BotFramework-WebChat/issues/1774). Add `styleSetOption` to allow word break. Default to `break-word`, by [@corinagum](https://github.com/corinagum) in PR [#1832](https://github.com/Microsoft/BotFramework-WebChat/pull/1832)
+- Fix [#1847](https://github.com/Microsoft/BotFramework-WebChat/issues/1847). Bump react-say, which adds babel-runtime dependency, by [@corinagum](https://github.com/corinagum) in PR [#1849](https://github.com/Microsoft/BotFramework-WebChat/pull/1849)
 
 ## [4.3.0] - 2019-03-04
 

--- a/packages/component/package.json
+++ b/packages/component/package.json
@@ -52,7 +52,7 @@
     "react-dictate-button": "^1.1.3",
     "react-film": "~1.1.2",
     "react-redux": "^5.0.7",
-    "react-say": "^1.1.1",
+    "react-say": "^1.1.2-master.453bced",
     "react-scroll-to-bottom": "~1.3.1",
     "redux": "^4.0.0",
     "sanitize-html": "^1.18.2",

--- a/packages/playground/package.json
+++ b/packages/playground/package.json
@@ -26,7 +26,7 @@
     "react-dictate-button": "1.1.3-master.97bc9a2",
     "react-dom": "^16.4.1",
     "react-redux": "^5.0.7",
-    "react-say": "1.0.1-master.1d32161",
+    "react-say": "1.1.2-master.453bced",
     "react-scripts": "2.1.1",
     "redux": "^4.0.0",
     "rxjs": "5",

